submissions/xT0PERx/html-movie-seat-booking/main.css

```diff
@@ -90,18 +86,17 @@ html {
 .placement {
   display: grid;
   grid-template-rows: repeat(6, [row] 1fr);
-  background-image: url(/Images/background.jpg);
+  background-image: url(./Images/background.jpg);
   width: 100%;
   height: 925px;
-  padding-right: 20px;
   background-repeat: no-repeat;
 }
 
 .screen {
   display: grid;
```

Comment:
In this case,  `display: grid` is excess. Try to turn off it in Dev Tools and see the result.)